### ISSUE TITLE
Can't resolve 'react/jsx-runtime.js' when using with storybook-addon-designs

### ISSUE THREAD

**Issue by pf1gura** [pocka/figspec#17]
### Issue
Not sure if this repo is right for the issue, but when following https://storybook-addon-designs-f8z5gt8iw.vercel.app/?path=/story/docs-figma-figspec-readme--page, my storybook instance crashes with
```
ERROR in ./node_modules/@figspec/react/esm/es2015/index.js
Module not found: Error: Can't resolve 'react/jsx-runtime.js' in '/Users/pf1gura/Projects/Redacted/node_modules/@figspec/react/esm/es2015'
```

### Possible solutions that did not work
I already looked online and tries few solutions, bot nothing worked so far.
I tried adding those aliases to webpack config
```
config.resolve.alias['react/jsx-runtime'] = 'react/jsx-runtime.js';
config.resolve.alias['react/jsx-dev-runtime'] = 'react/jsx-dev-runtime.js';
```

### Dependencies
```
"storybook-addon-designs": "^6.0.0-alpha.1",
"@figspec/components": "^0.1.3",
"@figspec/react": "^0.1.4",
"@storybook/angular": "^6.1.11",
```

**Comment by pocka** [comment]
Probably this package, especially on [the JSX transformation](https://github.com/pocka/figspec/blob/master/packages/react/tsconfig.json#L6). I have no idea why I choose React17 JSX transform...

**Comment by pocka** [comment]
The patch was shipped in v0.1.5.

**Comment by pf1gura** [comment]
It works now, thanks for quick action!
